internal/app/repository/user_repository_test.go

```diff
@@ -0,0 +1,72 @@
+package repository_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/model"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/repository"
+	"github.com/afernandowex/ps-tag-onboarding-go/internal/app/repository/mocks"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"gorm.io/gorm"
+)
+
+func TestUserRepository_FindByID(t *testing.T) {
```

Comment:
You dont have unit tests for save or exists

Reply:
I think using mockery is not a good idea for gorm unit tests. The recommended way as far as I read online is to use sqlmock instead. I've converted the repository tests to sqlmock instead of using mockery.